libfreerdp/core/server.c

```diff
@@ -1578,12 +1578,9 @@ BOOL WINAPI FreeRDP_WTSVirtualChannelWrite(HANDLE hChannelHandle, PCHAR Buffer,
 		WINPR_ASSERT(channel->client);
 		context = channel->client->context;
 		WINPR_ASSERT(context);
-		WINPR_ASSERT(context->settings);
 		while (Length > 0)
 		{
-			const UINT32 VirtualChannelChunkSize =
-			    freerdp_settings_get_uint32(context->settings, FreeRDP_VirtualChannelChunkSize);
-			s = Stream_New(NULL, VirtualChannelChunkSize);
+			s = Stream_New(NULL, 1600);
```

Comment:
sure the magic number or better `CHANNEL_CHUNK_LENGTH`?

Reply:
Did add a define for this. I did not choose `CHANNEL_CHUNK_LENGTH` though, since that is the term used in `[MS-RDPBCGR]` (for SVCs).
Since `[MS-RDPEDYC]` does not define a name for the 1600 bytes, I chose here `DVC_MAX_DATA_PDU_SIZE` (to differentiate between the value for all SVCs and the one for DVCs). I hope that is okay.